### PR TITLE
Fix BinaryenRemoveEvent naming in binaryen-c.h

### PR DESCRIPTION
**Pull request by dcodeIO**
As [discovered](https://travis-ci.org/AssemblyScript/binaryen.js/jobs/544330656#L5577) by the binaryen.js build bot. Also enables `ERROR_ON_UNDEFINED_SYMBOLS=1` in build-js.sh to catch what can be caught (not sure if it would catch this exact scenario) early.

### CODE REVIEW

**Comment by kripken** [comment]
Thanks @dcodeIO!

File: build-js.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ EMCC_ARGS="$EMCC_ARGS -s NO_FILESYSTEM=0"
 EMCC_ARGS="$EMCC_ARGS -s WASM=0"
 EMCC_ARGS="$EMCC_ARGS -s ERROR_ON_UNDEFINED_SYMBOLS=1"
 EMCC_ARGS="$EMCC_ARGS -s BINARYEN_ASYNC_COMPILATION=0"
-# TODO: enable this (need nearbyint in emscripten tag) EMCC_ARGS="$EMCC_ARGS -s ERROR_ON_UNDEFINED_SYMBOLS=1"
+EMCC_ARGS="$EMCC_ARGS -s ERROR_ON_UNDEFINED_SYMBOLS=1"
```
Comment (sbc100) [inline build-js.sh]: This is (a) already two lines above and (b) the default, so these lines should both be removed.

Comment (dcodeIO) [reply build-js.sh]: Nice catch, totally missed that. Did the default of `ERROR_ON_UNDEFINED_SYMBOLS` change recently? Iirc it didn't use to be enabled by default.

Comment (sbc100) [reply build-js.sh]: It changed a while ago.  Looks like v1.38.13

**Pull request merged**